src/macropad/index.html

```diff
@@ -0,0 +1,175 @@
+            <li>4x M3 x 6mm screws</li>
+            <li>4x M3 nuts</li>
+          </ul>
+        <li>
+          1x 5v Arduino Pro Micro
+        </li>
+        <li>
+          6x 1N4148 Diodes
+        </li>
+        <li>
+          4x Rubber Bumpers/Feet
+        </li>
+        <li>
+          6x PCB Mount Switches
+        </li>
+        <li>
+          6x 1u Keycaps
+        </li>
+      </ul>
+
+      <h2 id="build_guide">Build Guide</h2>
+
+      <h3>Tools</h3>
+      <p>You'll need</p>
+      <ul>
+        <li>A soldering iron and solder</li>
+        <li>A phillips screwdriver</li>
+        <li>Flush cutters</li>
+      </ul>
+
+      <h3>Assembly</h3>
+      <h4>Diodes</h4>
+      <p>Bend your diodes as shown</p>
+      <figure>
+        <img src="bentdiode.jpg" width="600"/>
+      </figure>
+      <p>
+        Place your diodes with the black stripe on the diode facing the white
+        stripe on the PCB, slightly bend the legs to prevent the diode from
+        falling out.
+      </p>
+      <figure>
+        <img src="placeddiode.jpg" width="600"/>
+        <img src="diodes.jpg" width="600"/>
+      </figure>
+      <p>Solder the diodes, and trim the legs with flush cutters.</p>
+      <figure>
+        <img src="trimmeddiodes.jpg" width="600"/>
+      </figure>
+      <h4>Switches & Pro Micro</h4>
+      <p>Place and solder the lower four switches</p>
+      <figure>
+        <img src="fourswitches.jpg" width="600"/>
+      <figure>
+      <p>
+        Tack on the two pin headers, fiddle with them until they're straight
+        and the pro micro fits on.
+      </p>
+      <figure>
+        <img src="tackedheader.jpg" width="600"/>
+        <img src="testfit.jpg" width="600"/>
+      <figure>
+      <p>Solder the pin header.</p>
+      <figure>
+        <img src="solderedheaders.jpg" width="600"/>
+      </figure>
+      <p>Solder the top two switches.</p>
+      <figure>
+        <img src="twoswitches.jpg" width="600"/>
+      </figure>
+      <p>Solder the Pro Micro onto the headers</p>
+      <figure>
+        <img src="solderedpromicro.jpg" width="600"/>
+      </figure>
+      <h4>Flash the Pro Micro</h4>
+      <p style="font-family: monospace;">
+        git clone git@github.com:qmk/qmk_firmware.git<br />
+        cd qmk_firmware<br />
+        util/linux_install.sh <br />
+        sudo make pdxkbc:default:avrdude<br />
+        # short the RST and GND pins on the pro micro when prompted
+      </p>
+      <p>Also see the github repo in the <a href="#links">Links</a> section.</p>
+      <h4>Test the board</h4>
+      <p>The default keymap is:</p>
+      <p style="font-family: monospace;">
+        const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = { [0] =
+        LAYOUT( PDXKBCREDDIT, PDXKBCDISCORD, BADGELIFE, HACKTHEPLANET,
+        KC_VOLU, KC_VOLD ), };
+      </p>
+      <h4>Finish Assembly</h4>
+      <p>Bolt and screw on the spacers, install the bumpers. You're done!</p>
+
+    <footer>
```

Comment:
Not sure whats wrong with at at a glance, but the footer is not the correct size. I'm guessing you didn't match the HTML structure of this page 100% correctly.